website/data/plugin-testing-nav-data.json

```
@@ -0,0 +1,39 @@
+[
+  { "heading": "Testing" },
+  { "title": "Overview", "path": "" },
+  { "title": "Migrate to the Testing Module", "path": "migrating" },
```

Nit: If we want to follow the same convention as the framework documentation

```suggestion
  { "title": "Migrating from SDK", "path": "migrating" },
```